cross/rackspace-novaclient/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME = rackspace-novaclient
```

Comment:
Are you sure this module needs cross-compiling? According to the source code, there's not much to it.